OpenRA.Mods.Common/Activities/CaptureActor.cs

```diff
@@ -23,15 +23,17 @@ public class CaptureActor : Enter
 		readonly Capturable capturable;
 		readonly Captures[] captures;
 		readonly Health health;
+		readonly WDist captureRange;
```

Comment:
> OpenRA.Mods.Common/Activities/CaptureActor.cs(26,18): error CS0414: Warning as Error: The private field `OpenRA.Mods.Common.Activities.CaptureActor.captureRange' is assigned but its value is never used

Since the actual handling happens in Enter, you don't need this here.

Reply:
Removed